<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>2-legged tokener</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>6</y>
      <width>1000</width>
      <height>341</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>359</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="handleWidth">
     <number>11</number>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="client_id_label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>65</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Client ID:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="client_id_input">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="client_secret_label">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>65</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Client Secret:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="client_secret_input">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="set_secrets_btn">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Set Secrets</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="get_token_btn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Get New Token</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Scopes</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <property name="topMargin">
          <number>13</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="sizeConstraint">
              <enum>QLayout::SetFixedSize</enum>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="sizeConstraint">
                <enum>QLayout::SetFixedSize</enum>
               </property>
               <item>
                <widget class="QCheckBox" name="data_read_check">
                 <property name="text">
                  <string>data::read</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="data_write_check">
                 <property name="text">
                  <string>data::write</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="data_create_check">
                 <property name="text">
                  <string>data::create</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="data_search_check">
                 <property name="text">
                  <string>data::search</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QCheckBox" name="bucket_create_check">
                 <property name="text">
                  <string>bucket::create</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="bucket_read_check">
                 <property name="text">
                  <string>bucket::read</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="bucket_update_check">
                 <property name="text">
                  <string>bucket::update</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="bucket_delete_check">
                 <property name="text">
                  <string>bucket::delete</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="topMargin">
              <number>15</number>
             </property>
             <item>
              <widget class="QCheckBox" name="code_all_check">
               <property name="text">
                <string>code::all</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="account_write_check">
               <property name="text">
                <string>account::write</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="account_read_check">
               <property name="text">
                <string>account::read</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="user_profile_check">
               <property name="text">
                <string>user-profile:read</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="token_table">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>770</width>
       <height>340</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>770</width>
       <height>340</height>
      </size>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
